### ISSUE TITLE
Train time and Train loss

### ISSUE THREAD

**Issue by kkkmax** [tsing90/pytorch_semantic_human_matting#10]
@tsing90  thanks!
I prepared the data according to the description of your Readme file, re-prepared the data, 180 DIM images and my own 880 collected images were processed into four-channel (RGBA) images, currently training, ask a few Questions about model training:
1) How long have you trained T-Net training? How much do you think the loss will fall to get a robust T-Net model?
2) How long did it take to train M-net, and how long did you train in end-to-end mode?

**Comment by tsing90** [comment]
1. my datasets are larger than yours, it took 50-100 epochs to converge, if you add more weights on unknown area, it can be faster;
2. m-net is similar; but my end-to-end was not stable, and I haven't figured out the reason, so currently I haven't finished yet.
